src/Tribe/JSON_LD/Event.php

```diff
@@ -59,8 +74,9 @@ public function get_data( $posts = null, $args = array() ) {
 			$event_tz_string = get_post_meta( $post_id, '_EventTimezone', true );
 			$tz_mode         = tribe_get_option( 'tribe_events_timezone_mode', 'event' );
 			$tz_string       = $event_tz_string && $tz_mode === 'event' ? $event_tz_string : Tribe__Events__Timezones::wp_timezone_string();
-			$data->startDate = Tribe__Events__Timezones::to_utc( tribe_get_start_date( $post_id, true, Tribe__Date_Utils::DBDATETIMEFORMAT ), $tz_string, 'c' );
-			$data->endDate   = Tribe__Events__Timezones::to_utc( tribe_get_end_date( $post_id, true, Tribe__Date_Utils::DBDATETIMEFORMAT ), $tz_string, 'c' );
+
+			$data->startDate = self::local_timezone( Tribe__Events__Timezones::to_utc( tribe_get_start_date( $post_id, true, Tribe__Date_Utils::DBDATETIMEFORMAT ), $tz_string, 'c' ), $event_tz_string );
```

Comment:
Let's add a comment above the line, something like:

```php
// @todo simplify to `tribe_get_start_date( $post_id, true, 'c' )`  once #xxxxx is resolved
```

Where #xxxxx should be the actual bug number (re `tribe_get_start_date()` / `tribe_format_date()` losing the timezone context) as discussed.